Moving / migrating SC instances to another server (#5047)
Particular/docs.particular.net: servicecontrol/installation.md

The ServiceControl queue name is required for configuring the endpoint plugins. See [Install Heartbeats Plugin](/monitoring/heartbeats/install-plugin.md) and [Install Custom Checks Plugin](/monitoring/custom-checks/install-plugin.md) for more information.

## Moving / Migrating

> Migrating would consist of:

Reviewer: ```suggestion
In order to migrate an instance of ServiceControl to another host, the following must be performed:
```

> - Stopping and disabling ServiceControl instance type services in the Windows Service Manager for current instances to halt message ingestion.
> - Installing and configurating the ServiceControl instance types on the new server
> - Ensure the services on the new location are stopped.
> - Copy the data from the previous to the new server (moving is not advised).
> - Start the services

Reviewer: ```suggestion
1. Stop and disable the current instance of ServiceControl in the Windows Service Manager.
1. Install and configure a new instance of ServiceControl on the new server.
1. Stop the newly installed instance of ServiceControl.
1. Copy the data from the previous instance to the new instance server (moving is not advised).
1. Start the new service.
1. Remove the old instance of ServiceControl.
```

> NOTE: If Databases are very large no downtime is possible then consider to [scale-out ServiceControl via remote instances](/servicecontrol/servicecontrol-instances/remotes).

Reviewer: ```suggestion
NOTE: If the database of the instance being migrated is very large and no downtime can be tolerated, then consider to [scale-out ServiceControl via remote instances](/servicecontrol/servicecontrol-instances/remotes).
```

Ensure that:

> - You use the same installer as the current version. Database schemas are not guaranteed to be compatible between different versions
> - Optionally, first update the current installation to the latest versions.

Reviewer: ```suggestion
- First update the current instance to the latest version.
- Optionally, use the same installer as the current version. Database schemas are not guaranteed to be compatible between different versions.
```

- Optionally, [script ServiceControl installations via powershell](/servicecontrol/installation-powershell) instead of running installers manually

> ServiceControl instance types configuration settings are accessible via the  Service Control Management Utility user-interface or by navigation to the configurationg files on the file-system stored in `ServiceControl.exe.config`, `ServiceControl.Audit.exe.config`, and `ServiceControl.Monitoring.exe.config`. 

Reviewer: ```suggestion
ServiceControl configuration settings are accessible via the  Service Control Management Utility user-interface or by navigating to the configuration files on the file-system stored in `ServiceControl.exe.config`, `ServiceControl.Audit.exe.config`, and `ServiceControl.Monitoring.exe.config`. 
```

> Note: If previously ServiceControl was installed via the ServiceControl Management Utility then all instance types are installed on a single machine. If the system has a considerable load or has a very large retention period consider installing just a single instance type on a (virtual) server to scale out. This can b done via Powershell.

Reviewer: ```suggestion
Note: If ServiceControl was previously installed via the ServiceControl Management Utility then all instances are installed on a single machine by default. If the system has considerable load or has a large retention period, consider installing a single instance type on a server to scale out. This can be done via Powershell.
```
